workflows/diagnostics/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -615,13 +615,23 @@ def add_catalog_and_verification_arguments(parser: ArgumentParser):
     )
 
 
+def add_time_range_arguments(parser: ArgumentParser):
+    parser.add_argument(
+        "--start-date", type=str, help="Start date for diagnostics", default=""
+    )
+    parser.add_argument(
+        "--end-date", type=str, help="End date for diagnostics", default=""
```

Comment:
suggestion: `None` happens to be a valid input as a `start` and/or `end` to `slice`, which means no bound (e.g. `slice(None, None)` returns the full data).  Using it as the default value for `--start-date` and `--end-date` would remove the need for the conditional logic in `evaluation_pair_to_input_data`, and would allow for the case of setting just one of the dates (e.g. maybe one only needs to set an `--end-date` bound).  

```suggestion
        "--start-date", type=str, help="Start date for diagnostics", default=None
    )
    parser.add_argument(
        "--end-date", type=str, help="End date for diagnostics", default=None
```

Reply:
Great suggestion, thank you!